src/inspector_agent.cc

```diff
@@ -72,15 +72,23 @@ std::string StringViewToUtf8(const StringView& view) {
   }
 
   // CHAKRA-TODO: Figure out what to do here
-  assert(false);
-
   const uint16_t* source = view.characters16();
+  const size_t length = view.length();
+  std::string result;
+
+  for (size_t i = 0; i < length; i++) {
+    uint16_t wCh = source[i];
+    assert((wCh & 0xFF80) == 0);
+
+    result.append(1, static_cast<char>(wCh));
+  }
+
   ////const UChar* unicodeSource = reinterpret_cast<const UChar*>(source);
   ////static_assert(sizeof(*source) == sizeof(*unicodeSource),
   ////              "sizeof(*source) == sizeof(*unicodeSource)");
 
-  size_t result_length = view.length() * sizeof(*source);
-  std::string result(result_length, '\0');
```

Comment:
I can delete it, but the hope was that if we can enable ICU in the build that we would be able to just uncomment the code and enable Unicode support.